test/userapp/index.js

```diff
@@ -398,7 +441,12 @@ const addRequestHeader = (request, z, bundle) => {
   could be useful for APIs that always return 200 even on errors.
  */
 const changeStatusOnErrorResponses = (response) => {
-  const isJsonResponse = response.getHeader('Content-Type').match(/^application\/json/);
+  const contentType = response.getHeader('Content-Type');
+  if (!contentType) {
+    return response;
+  }
+
+  const isJsonResponse = contentType.match(/^application\/json/);
```

Comment:
Probably don't need a regex here. Right now, it'll match `application/jsonwhatever`, when you really just want to verify that it's `=== 'application/json`